daemon/top_unix.go

```diff
@@ -79,6 +79,10 @@ func parsePSOutput(output []byte, procs []uint32) (*container.ContainerTopOKBody
 	if pidIndex == -1 {
 		return nil, fmt.Errorf("Couldn't find PID field in ps output")
 	}
+	//remove appended pid in first cloumn
```

Comment:
* a space after `//` would be nice to have
* `cloumn` -> `column`

Reply:
Thanks